### PR TITLE
Datago-658

### PR DESCRIPTION
**Pull request by alyrasmy**
-Pulled in latest template changes
-Resolved merge conflict
-Added svgs and changed tutorials to use those svgs
-Removed extra png image in the tutorials
-Fixed issues with samples params
-Change samples params to the following proposed format: ./sampleName HOST:PORT USER@MSGVPN PASSWORD
-Removed any dataGo reference and updated it to solace Cloud
-Samples clearly outline how to use solace Cloud in the tutorials (only for solace cloud version (JEKYLL_ENV=solaceCloud)

**To test:**
* JEKYLL_ENV=solaceCloud jekyll serve (for solace cloud website version)
*  jekyll serve (for Solace  dev website version

screen shot 2017-10-17 at 6 12 06 pm
screen shot 2017-10-17 at 6 12 20 pm
screen shot 2017-10-17 at 6 12 43 pm
screen shot 2017-10-17 at 6 13 18 pm
screen shot 2017-10-17 at 6 14 22 pm

### CODE REVIEW

**Comment by alyrasmy** [comment]
screen shot 2017-10-18 at 4 28 39 pm
screen shot 2017-10-18 at 4 30 06 pm

File: docs/_tutorials/confirmed-delivery.md [hunk 1]
```diff
@@ -1,63 +1,44 @@
...
 *   You are familiar with Solace [core concepts]({{ site.docs-core-concepts }}){:target="_top"}.
-*   You have access to a running Solace message router with the following configuration:
-    *   Enabled message VPN configured for guaranteed messaging support.
-    *   Enabled client username.
+*   You have access to Solace messaging with the following configuration details:
+    *   Connectivity information for a Solace message-VPN configured for guaranteed messaging support
+    *   Enabled client username and password
     *   Client-profile enabled with guaranteed messaging permissions.
 
-Note that one simple way to get access to a Solace message router is to start a Solace VMR load [as outlined here]({{ site.docs-vmr-setup }}){:target="_top"}. By default the Solace VMR will with the “default” message VPN configured and ready for guaranteed messaging. Going forward, this tutorial assumes that you are using the Solace VMR. If you are using a different Solace message router configuration adapt the tutorial appropriately to match your configuration.
```
Comment (mdspielman) [inline docs/_tutorials/confirmed-delivery.md]: Can we keep this text in the "non-solace cloud" builds?

File: docs/_tutorials/topic-to-queue-mapping.md [hunk 2]
```diff
@@ -23,15 +23,15 @@ If you have a durable queue named `Q`, it will receive messages published direct
...
-    *   Enabled client username.
-    *   Client-profile enabled with guaranteed messaging permissions.
 *   You understand the basics introduced in [Persistence with Queues]({{ site.baseurl }}/persistence-with-queues)
+*   You have access to Solace messaging with the following configuration details:
+    *   Connectivity information for a Solace message-VPN configured for guaranteed messaging support
+    *   Enabled client username and password
+    *   Client-profile enabled with guaranteed messaging permissions.
 
-Note that one simple way to get access to a Solace message router is to start a Solace VMR load [as outlined here]({{ site.docs-vmr-setup }}){:target="_top"}. By default the Solace VMR will with the “default” message VPN configured and ready for guaranteed messaging. Going forward, this tutorial assumes that you are using the Solace VMR. If you are using a different Solace message router configuration adapt the tutorial appropriately to match your configuration.
-
```
Comment (mdspielman) [inline docs/_tutorials/topic-to-queue-mapping.md]: Same here. Can we keep this text for non solace cloud.

**Comment by alyrasmy** [comment]
screen shot 2017-10-19 at 11 07 23 am

**Pull request merged**